.github/workflows/hyperion_check-openapi.yml

```
@@ -0,0 +1,92 @@
+          version: 2.1.1
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+          virtualenvs-path: .venv
+          installer-parallel: true
+
+      - name: Load cached venv
+        id: cached-poetry-dependencies
+        uses: actions/cache@v4
+        with:
+          path: .venv
+          key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
```

actionlint (1.7.7)

property "python-version" is not defined in object type {}

(expression)